### ISSUE TITLE
Terminal view is not accessible for the screen readers

### ISSUE THREAD

**Issue by ilyapashuk** [connectbot/connectbot#1552]
## Bug description

The ConnectBot's terminal view is not accessible for Talkback and other screen reader software

## Steps to reproduce

1. install (if needed) and activate the Talkback Screen reader
2. launch ConnectBot and connect to the any server (concrete server and protocol do not matter)
3. when connection is opened and the first output can be seen in the view, touch the terminal view to make Talkback to read it's contents;
4. hear nothing instead of the output text
5. do doubletap with long press on the view, to start a selection mode
6. touch the view again
7. hear the output content
8. exit selection mode with back button
9. type something in the terminal to raise new output
10. touch the view
11. the old output will be spoken again, but not the new.
12. enter selection mode again
13. hear the updated output

## Expected behavior

screen reader should sea any new output immediately.

## Screenshots

not applicable, there is no visual information to present.

## Android device

 - Device: Samsung Galaxy A30
 - OS: Android 11
 - ConnectBot Version: 1.9.12

## Server information

not applicable, problem accurs with any server

## Additional context

It theams that, when the selection mode is started, some property of the view (may be "Text" or something like this) is updated with the current terminal contents. That update should be done automaticly on any terminal change, that will solve the problem.

Compare behavior with the Termux terminal emulator.

**Comment by kruton** [comment]
I think this is because of the overlay text view used for selecting text. Basically the overlay text view is doing the correct thing, but the terminal emulation is not. I'll see what I can do to fix it.

**Comment by kruton** [comment]
@ilyapashuk I'm working on rewriting the Terminal component as a separate library. I'd like to make it accessible because it might be used by others. Could you contribute to the discussion at https://github.com/connectbot/connectbot/discussions/1625 ?

**Comment by kruton** [comment]
I've completed the first pass of accessibility for the new terminal component. I hope to have it in the beta build tonight.

Here are the features:

-  creates a shadow list of text that is explorable with touch exploration or swiping between UI elements. Explore line by line by default, but you can use TalkBack to switch to word and character granularity 

- Custom accessibility commands:
  - Enter review mode to enable going into the scroll back that is off screen without affecting cursor position
  - Go to previous command prompt and command output. This requires shell to have OSC 133 integration. I have a document describing how to implement this in bash and zsh.
  - Copy line. I want to expand this to be able to start selection and copy multiple lines. I don't think I can get word or character granularity unfortunately.
